app/js/components/topics/topicRuleList/topicRuleList.jade

```diff
@@ -40,7 +40,7 @@
                 .sort-indicator(translate) Likelihood
               th.text-center.sortable(ng-class="sorter.getSortClass('rec_impact')", ng-click="sorter.sort('rec_impact')")
                 .sort-indicator(translate) Impact
-              th.text-center.sortable(ng-class="sorter.getSortClass('severityNum')", ng-click="sorter.sort('severityNum')")
+              th.text-center.sortable(ng-class="sorter.getSortClass('-severityNum')", ng-click="sorter.sort('-severityNum')")
```

Comment:
Looks like this actually breaks the ordering?

![screenshot_20171205_155757](https://user-images.githubusercontent.com/167352/33613549-2f3cf380-d9d5-11e7-83c4-571ff1cfa9af.png)

The arrow shows ascending whereas the severity of rules is descending